test_server.py

```diff
@@ -0,0 +1,31 @@
+import server
+import socket
+import pytest
+from multiprocessing import Process
+
+ADDR = ('127.0.0.1', 8080)
+
+
+@pytest.yield_fixture()
+def server_setup():
+    process = Process(target=server.start_server)
+    process.daemon = True
+    process.start()
+    yield process
+
+
+@pytest.fixture()
+def client_setup():
+    client = socket.socket(
+        socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP
+    )
+    client.connect(ADDR)
+    return client
+
+
+def test_response_ok(server_setup):
+    assert '200 OK' in server.response_ok()
```

Comment:
This test is necessary, but it is not sufficient.  Just because `200 OK` is in a string, does not make that string a well-formed HTTP response.  What other conditions must you test for?